src/auth/screens/events.screen.js

```diff
@@ -526,6 +547,8 @@ class Events extends Component {
           onRefresh={this.getUserEvents}
           refreshing={isPendingEvents}
           keyExtractor={this.keyExtractor}
+          onScroll={this.changeTabBarVisibility}
+          onEndReachedThreshold={1200}
```

Comment:
Can you move this `1200` to a constant with a meaningful name? Also why `1200` instead of `1000` for example?